gaia-6000/genesis.json

```diff
@@ -400,146 +416,151 @@
                         }
                     ]
                 },
-                {
-                    "address": "A28BD79FBE0B07712B52BC064FCEC0C084B4EF84",
-                    "coins": null
-                },
                 {
                     "address": "A323EC45243D600204BA3D298E3C20322D08C84C",
-                    "coins": null
+                    "coins": [
+                        {
+                            "denom": "steak",
+                            "amount": 50
+                        },
+                        {
+                            "denom": "nuevaxToken",
+                            "amount": 1000
+                        }
+                    ]
                 },
                 {
                     "address": "A9A3DADDC8BFFAD52BA51C8F4F2E9F62709412DC",
-                    "coins": null
-                },
-                {
-                    "address": "AB30D2D5B4EFA4652917822F80603ECCAC674AB7",
                     "coins": [
                         {
-                            "denom": "poldsamToken",
-                            "amount": 1
+                            "denom": "steak",
+                            "amount": 50
+                        },
+                        {
+                            "denom": "dokiaToken",
+                            "amount": 1000
                         }
                     ]
                 },
                 {
                     "address": "ABB0A52177EA1E6FD932E46438E2C3C6538FC6BA",
-                    "coins": null
-                },
-                {
-                    "address": "B63112E0A97226A94F0A2B40BBD68947FFFC44A1",
                     "coins": [
                         {
                             "denom": "steak",
-                            "amount": 1
+                            "amount": 50
+                        },
+                        {
+                            "denom": "gnitestToken",
+                            "amount": 1000
                         }
                     ]
                 },
                 {
                     "address": "B6834D914FE50F0C743E6A012AB20438CFADFB95",
-                    "coins": null
-                },
-                {
-                    "address": "B73A978396D04460FF4BE10E0AACD989CD648F82",
                     "coins": [
                         {
                             "denom": "steak",
-                            "amount": 2
+                            "amount": 50
+                        },
+                        {
+                            "denom": "forboleToken",
+                            "amount": 1000
                         }
                     ]
                 },
                 {
                     "address": "B75C2C4309475C91E8DE271BC52CBAC408365B83",
```

Comment:
I'd like to separate the tokens between my validator owner and faucet addresses. Can we do that here, or should I manually send the coins to the faucet address later? 

Validator Owner Address: B75C2C4309475C91E8DE271BC52CBAC408365B83
Faucet Address: FD30D5C983FFEDEC069C3DDFCF270E41A556A86E